spec/dummy/config/boot.rb

```diff
@@ -3,6 +3,8 @@
 
 ENV["AVO_IN_DEVELOPMENT"] = "1"
 
+ENV["SECRET_KEY_BASE"] = "130b8d0a74b5b73bfb2d0505c3de8250"
```

Comment:
Please remind me, why do we need the `ENV["SECRET_KEY_BASE"]`?

Reply:
For features that uses the encryption service to work, for example select all that encrypts the `@query`

Reply:
Oh, yeah. Makes sense. I was thinking about something else.
GOod to merge!

Reply:
Without this variable that features would not work properly on tests or dev. Another solution would be to configure it on development and test credential files.

Anyway `130b8d0a74b5b73bfb2d0505c3de8250` is just a random string, it can be anything. What is important is to use on decryption the same string used on encryption